src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/UpdateDocumentById.cs

```diff
@@ -223,8 +219,42 @@ await _sqlAction.DeleteReferencesByDocumentUuid(
                         }
                     }
 
+                    var cascadingUpdateResults = await _sqlAction.CascadeUpdates(
+                        updateRequest.ResourceInfo.ResourceName.Value,
+                        documentId,
+                        documentPartitionKey.Value,
+                        updateRequest.DocumentInfo,
+                        connection,
+                        transaction,
+                        traceId
+                    );
+
+                    await cascadeUpdates(cascadingUpdateResults);
+
                     return new UpdateResult.UpdateSuccess(updateRequest.DocumentUuid);
 
+                    // Recursively call CascadeUpdates until the results are exhausted
+                    async Task cascadeUpdates(List<CascadingUpdateResult> results)
```

Comment:
Rename to something like `recursivelyCascadeUpdate()` - right now it's the same name as `SqlAction.CascadeUpdates()`